resources/dicts/events/death/general/leader.json

```diff
@@ -967,7 +967,7 @@
             "dislike",
             "neg_respect"
         ],
-        "death_text": "m_c was pushed under a monster, and lost a life.",
+        "death_text": "m_c {VERB/m_c/were/was} pushed under a monster, and lost a life.",
```

Comment:
I'm not sure pronoun tagging is needed here, if m_c is a proper name it would always be "was", wouldn't it?